lib/ecwid_api/error.rb

```diff
@@ -8,7 +8,6 @@ def initialize(response)
 
     def get_response_info(response)
       url = response.env.url
-      url.query = url.query.sub(/token=[^&]*/, 'token=**SECRET**')
```

Comment:
We no longer have the token, aka `query`, set as we put `token` in the `headers` now.